src/index.ts

```diff
@@ -1,12 +1,79 @@
 import { Probot } from "probot";
+import fs from 'fs';
+const k8s = require('@kubernetes/client-node');
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
+const k8sContext = kc.getCurrentContext();
+const namespace = (() => {
+  if (k8sContext === 'inClusterContext') {
+    return fs.readFileSync('/var/run/secrets/kubernetes.io/serviceaccount/namespace', 'utf8');
+  } else {
+    return k8sContext.split('/')[0];
+  }
+})();
+console.log(`Operating in the namespace: ${namespace}`);
 
 export = (app: Probot) => {
+
+  // Respond to an issue opened
   app.on("issues.opened", async (context) => {
     const issueComment = context.issue({
       body: "Thanks for opening this issue!",
     });
     await context.octokit.issues.createComment(issueComment);
   });
+
+  // Respond to the GitHub app installation
+  app.on("installation.created", async (context) =>{
+
+    // Get the installation token  and expiry time from the installation
+    const appAuth: any  = await context.octokit.auth({ type: "installation" });
+    // console.log(appAuth);
```

Comment:
let's remove the commented out code, please

Reply:
thanks removed it.